apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx

```diff
@@ -30,6 +30,7 @@ export default class Results extends React.Component {
       if (['scale', 'singleSelect'].includes(question['answer_type'])) {
         return (
           <SingleChoiceResponses
+            perFacilitator={section === "facilitator"}
```

Comment:
I'd suggest making this `section === "facilitator" && permission.hasAny('workshop_organizer', 'program_manager', 'workshop_admin')`. Facilitators should never see per facilitator breakdown

Reply:
Should they never see it, or should they only see their own responses?

Reply:
They should only see their own responses if they are facilitators without other permissions

Reply:
As it currently stands, facilitators without other permissions will see the view on the right

![image](https://user-images.githubusercontent.com/244100/46322895-4ea9f300-c5a0-11e8-90b7-dc8d0b2edb6f.png)

Reply:
I think this view still works. Facilitators know that they can't see their co-facilitators scores, so we can explain more in their workshop dashboard guide why the math here may seem off.

Reply:
I can't tell from this example because the facilitators got the same responses. But will the total percentage only reflect the percentage of the 1 facilitator or both facilitators? Assuming the former?

Reply:
Hah, yeah, my local sample data wasn't particularly well-thought-out.

Your assumption is correct; the only data received by the right-hand view is data for the single facilitator viewing it, so all calculations are based on that.

Reply:
Ok, that makes sense so they can't calculate back and identity the other facilitator scores. I'll update in our workshop dashboard guides that this total percentage changes depending on if you're viewing as a facilitator or as a program manager (that sees all facilitator scores).